hack/lib/build/images.sh

```diff
@@ -120,7 +127,9 @@ function os::build::image::internal::docker() {
 	local extra_tag="${3-}"
 	local options=()
 
-	docker build ${OS_BUILD_IMAGE_ARGS:-} -t "${tag}" "${directory}"
+	if ! docker build ${OS_BUILD_IMAGE_ARGS:-} -t "${tag}" "${directory}"; then
+		return 1
```

Comment:
I'm not aware of any place we discriminate on specific non-zero return code but we can push up `$?` if you'd prefer